tests/TypeReconciliation/ConditionalTest.php

```diff
@@ -2839,6 +2839,15 @@ function b(B $_b): void {
                         echo "false";
                     }',
             ],
+            'BinaryDataIsPassedSeeIssue7771' => [
+                'code' => '<?php
```

Comment:
tests format changed between 4.x and master. removing `'code' => ` should fix the build